src/main/java/com/synopsys/integration/detect/workflow/blackduck/project/SyncProjectOperation.java

```diff
@@ -27,14 +28,14 @@ public SyncProjectOperation(ProjectService projectService) {
         this.projectService = projectService;
     }
 
-    public ProjectVersionWrapper sync(NameVersion projectNameVersion, ProjectGroupFindResult projectGroupFindResult, CloneFindResult cloneFindResult,
-        ProjectSyncOptions projectSyncOptions) throws DetectUserFriendlyException, IntegrationException {
-        ProjectSyncModel projectSyncModel = createProjectSyncModel(projectNameVersion, projectGroupFindResult, cloneFindResult, projectSyncOptions);
+    public ProjectVersionWrapper sync(NameVersion projectNameVersion, ProjectGroupFindResult projectGroupFindResult, CloneFindResult cloneFindResult, ProjectVersionLicensesFindResult projectVersionLicensesFindResult,
+                                      ProjectSyncOptions projectSyncOptions) throws DetectUserFriendlyException, IntegrationException {
```

Comment:
It looks like your formatter might be using the wrong indentation. 
Also if the line wraps with parameters, we should put them on separate lines.

Reply:
Yeah I don't know what's up with my formatter's indenting I'm using the one at https://sig-confluence.internal.synopsys.com/display/integration/Engineer+On-Boarding